File: src/androidTest/java/pt/joaomneto/titancompanion/TestEOTD.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;


@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestEOTD extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return FightingFantasyGamebook.EYE_OF_THE_DRAGON;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestEOTD.kt
package pt.joaomneto.titancompanion

import android.support.test.filters.LargeTest
import android.support.test.runner.AndroidJUnit4

import org.junit.runner.RunWith

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestEOTD : TestST() {

    override val gamebook = FightingFantasyGamebook.EYE_OF_THE_DRAGON
}